src/commands/mod.rs

```diff
@@ -208,6 +210,13 @@ struct ConfigListArgs {
     // TODO(#1047): Support ConfigArgs (--user or --repo).
 }
 
+/// Get the value of a given config option.
```

Comment:
An example would be nice, so it appears for `jj config get --help`

Reply:
It looks like it's not easy (possible?) to add code samples to the docstrings because clap will remove the intermediate newlines and unwrap the text. I wrote an inline example describing two invocations of `jj config` instead.

Reply:
Maybe you're looking for `#[command(verbatim_doc_comment)]` (which we use on `RebaseArgs`)

Reply:
@martinvonz thanks, updated.